Cargo.toml

```diff
@@ -12,3 +12,5 @@ members = [
     "rust-connectors/utils/fluvio-smartstream-map/"
 ]
 resolver = "2"
+[patch.crates-io]
+rumqttc = { git = "https://github.com/bytebeamio/rumqtt" }
```

Comment:
I think that we should remove Cargo.lock from .gitignore as per https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html and then add that to the repo